gridstatus/nyiso.py

```diff
@@ -476,6 +476,33 @@ def _download_nyiso_archive(self, date, end=None, dataset_name=None, filename=No
 
         return df
 
+    def get_capacity_prices(
+        self, date=pd.Timestamp.now(tz="US/Eastern"), verbose=False
```

Comment:
small python thing -- default arguments are only evaluated once when function is initiated. that means `date` will be the same value on every call. probably not a huge deal for this method since it only cares about month, but might as well change


```python
>>> import pandas as pd
def default_args_test(test=pd.Timestamp.now()):
    print(test)>>> def default_args_test(test=pd.Timestamp.now()):
...     print(test)
... 
>>> default_args_test()
2022-11-02 09:40:19.540002
>>> default_args_test()
2022-11-02 09:40:19.540002
```

you probably want to do something like this instead

```python
>>> def default_args_test(test=None):
...     if test is None:
...         test = pd.Timestamp.now()
...     print(test)
... 
>>> default_args_test()
2022-11-02 09:42:30.128012
>>> default_args_test()
2022-11-02 09:42:31.481343
```